ia-guide/ia-static.md

```diff
@@ -0,0 +1,344 @@
+# Information Architecture
+
+Let's go on an adventure into another world.
+This world is not entirely different to our own, but it might have some unexpected treasures - and pitfalls - to find.
+It's an adventure into the mind of your reader.
+They live in our world, but they don't always think like we do.
+And learning to think like a reader can be tricky!
+
+![A blue and green globe sitting on top of a book with a red cover](images/reader_world.svg "The readers' world")
+
+Have you ever been really good at something, and tried to show someone else how to do it?
+It's frustrating when they just don't seem to get it, even though it seems easy to you.
+Science tells us that the more you know about a topic, the less you are able to imagine what it's like to not know it.
+The technical name for this is the Curse of Knowledge, and it can make writing documentation hard.
+
+Writers combat the Curse of Knowledge by putting themselves in the shoes of the reader.
+And while this guide won't teach you to be a professional technical writer, it will definitely help you think about your reader, so that you can write better documentation.
+
+This guide is interactive, just click the option you like the most to move on.
```

Comment:
Should we get rid of this notion of interactivity in the static guide?

Reply:
Ah, I thought I'd gotten rid of all that. Good spot!